k-distribution/tests/regression-new/map-symbolic-tests/assignment-16-spec.k.out

```diff
@@ -0,0 +1,4 @@
+<k>
+  assignmentResult ( ( X |-> 1
+  MAP ) [ Y <- 2 ] [ Z <- 3 ] )
+</k>
```

Comment:
`M x |-> 1 z |-> 3 [y <- 2]` will be the fully evaluated term, for the record.

Reply:
Why? We don't know that `notBool Z in_keys(MAP)` is `true`, so how can we get `MAP Z |-> 3`?

Reply:
um, you're right.  i didn't think the case where `MAP` can be substituted by another map.

then, if there is no information about `MAP`, it is not possible to extend it, like `MAP  A |-> B`, which is the sound behavior.